lib/inventory_refresh/inventory_collection/scanner.rb

```diff
@@ -136,7 +130,7 @@ def build_parent_inventory_collection!
 
       def supports_building_inventory_collection?
         # Don't try to introspect ICs with custom query or saving code
-        return if arel.present? || custom_save_block.present?
+        return if !arel.nil? || custom_save_block.present?
```

Comment:
What is the point of this change?

Reply:
For some reason `arel.present?` started to actually do the query, which can be very expensive. (I was reviewing the sweep queries and found this)

Reply:
:scream: okay good change then haha